doc/modules/model_evaluation.rst

```diff
@@ -28,1615 +30,20 @@ model:
-
-.. math::
-
-  L_\text{Hinge}(y, w) = \max\left\{1 - wy, 0\right\} = \left|1 - wy\right|_+
-
-If there are more than two labels, :func:`hinge_loss` uses a multiclass variant
-due to Crammer & Singer.
-`Here <http://jmlr.csail.mit.edu/papers/volume2/crammer01a/crammer01a.pdf>`_ is
-the paper describing it.
-
-If :math:`y_w` is the predicted decision for true label and :math:`y_t` is the
-maximum of the predicted decisions for all other labels, where predicted
-decisions are output by decision function, then multiclass hinge loss is defined
-by:
-
-.. math::
-
-  L_\text{Hinge}(y_w, y_t) = \max\left\{1 + y_t - y_w, 0\right\}
-
-Here a small example demonstrating the use of the :func:`hinge_loss` function
-with a svm classifier in a binary class problem::
-
-  >>> from sklearn import svm
-  >>> from sklearn.metrics import hinge_loss
-  >>> X = [[0], [1]]
-  >>> y = [-1, 1]
-  >>> est = svm.LinearSVC(random_state=0)
-  >>> est.fit(X, y)
-  LinearSVC(C=1.0, class_weight=None, dual=True, fit_intercept=True,
-       intercept_scaling=1, loss='squared_hinge', max_iter=1000,
-       multi_class='ovr', penalty='l2', random_state=0, tol=0.0001,
-       verbose=0)
-  >>> pred_decision = est.decision_function([[-2], [3], [0.5]])
-  >>> pred_decision  # doctest: +ELLIPSIS
-  array([-2.18...,  2.36...,  0.09...])
-  >>> hinge_loss([-1, 1, 1], pred_decision)  # doctest: +ELLIPSIS
-  0.3...
-
-Here is an example demonstrating the use of the :func:`hinge_loss` function
-with a svm classifier in a multiclass problem::
-
-  >>> X = np.array([[0], [1], [2], [3]])
-  >>> Y = np.array([0, 1, 2, 3])
-  >>> labels = np.array([0, 1, 2, 3])
-  >>> est = svm.LinearSVC()
-  >>> est.fit(X, Y)
-  LinearSVC(C=1.0, class_weight=None, dual=True, fit_intercept=True,
-       intercept_scaling=1, loss='squared_hinge', max_iter=1000,
-       multi_class='ovr', penalty='l2', random_state=None, tol=0.0001,
-       verbose=0)
-  >>> pred_decision = est.decision_function([[-1], [2], [3]])
-  >>> y_true = [0, 2, 3]
-  >>> hinge_loss(y_true, pred_decision, labels)  #doctest: +ELLIPSIS
-  0.56...
-
-.. _log_loss:
-
-Log loss
---------
-
-Log loss, also called logistic regression loss or
-cross-entropy loss, is defined on probability estimates.  It is
-commonly used in (multinomial) logistic regression and neural networks, as well
-as in some variants of expectation-maximization, and can be used to evaluate the
-probability outputs (``predict_proba``) of a classifier instead of its
-discrete predictions.
-
-For binary classification with a true label :math:`y \in \{0,1\}`
-and a probability estimate :math:`p = \operatorname{Pr}(y = 1)`,
-the log loss per sample is the negative log-likelihood
-of the classifier given the true label:
-
-.. math::
+The scoring parameters and the various metric mentioned above are explained in detail in separate
+pages.
 
-    L_{\log}(y, p) = -\log \operatorname{Pr}(y|p) = -(y \log (p) + (1 - y) \log (1 - p))
+.. toctree::
```

Comment:
I'm not even sure it's necessary to display this. You can add the `:hidden:` option.